src/main/scala/cosas/ops/types/SerializeDenotations.scala

```diff
@@ -0,0 +1,44 @@
+package ohnosequences.cosas.ops.types
+
+import ohnosequences.cosas._, types._, typeSets._, fns._
+
+// TODO msg not found
+trait SerializeDenotations[Denotations <: AnyTypeSet, V]
+extends Fn2[Denotations, Map[String,V]] with
```

Comment:
- of course I can use it this way, I just don't understand what this `Map()` is for
- I really don't get your point. and if you won't give me an example of when this thing is crucial, better don't loose your time and just merge this, the rest is already fixed as I see.